play-scala-grpc-example/project/Dependencies.scala

```diff
@@ -0,0 +1,28 @@
+package play.scala.grpc.sample
+
+import sbt._
+import sbt.Keys._
+
+object Dependencies {
+
+  // Test libraries
+  val playVersion = play.core.PlayVersion.current
+
+  val playGrpcVersion = "0.8.1"
+  val CompileDeps = Seq(
+    play.sbt.PlayImport.guice,
+    "com.lightbend.play"      %% "play-grpc-runtime"   % BuildInfo.playGrpcVersion, 
+    "com.typesafe.akka"       %% "akka-discovery"      % "2.6.1", 
+    "com.typesafe.akka"       %% "akka-http"           % "10.1.11",
+    // Test Database
+    "com.h2database" % "h2" % "1.4.199"
+  )
+
+  val TestDeps = Seq(
+    "com.lightbend.play"      %% "play-grpc-scalatest" % BuildInfo.playGrpcVersion % Test, 
+    "com.lightbend.play"      %% "play-grpc-specs2"    % BuildInfo.playGrpcVersion % Test, 
+    "com.typesafe.play"       %% "play-test"           % playVersion     % Test, 
+    "com.typesafe.play"       %% "play-specs2"         % playVersion     % Test, 
+    "org.scalatestplus.play"  %% "scalatestplus-play"  % "5.0.0" % Test, 
+  )
+}
```

Comment:
I dislike this setup as (in my eyes) it needlessly splits the build into multiple files.  Perhaps there's value in it when you have a build as large as Play/Lagom, but within an sample project it seems to me even more needless.

That said, it's no blocker to the progress being landed with this!  Clear for landing!

Reply:
I can move it back, no problem